app/models/media.rb

```diff
@@ -96,6 +96,7 @@ def normalize_url
   end
 
   def follow_redirections
+    self.url = self.url.strip
```

Comment:
This command should be in the normalization function, not in the redirection function.